addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/UniFiController.java

```diff
@@ -105,6 +106,20 @@ public void logout() throws UniFiException {
         executeRequest(req);
     }
 
+    public void blockStation(UniFiClient client, boolean block, String siteName) throws UniFiException {
+        UniFiControllerRequest<UniFiClient[]> req = newRequest(UniFiClient[].class);
```

Comment:
Since we're not interested in the controller response, you can safely make this a `newRequest(Void.class)` request which will skip the JSON unmarshalling.

Reply:
Yes, the function now is a "void function".